src/js/techreport/index.js

```diff
@@ -25,17 +25,22 @@ class TechReport {
     this.initializePage();
     this.getAllMetricData();
     this.bindSettingsListeners();
+    this.initializeAccessibility();
   }
 
   // Initialize the sections for the different pages
   initializePage() {
+    console.log('initialize page');
+    this.updateStyling();
```

Comment:
Assume this isn't needed
```suggestion
    this.updateStyling();
```